mods/ts/weapons/superweapons.yaml

```diff
@@ -56,19 +56,19 @@ SuicideBomb:
 	Warhead@2Res: DestroyResource
 
 IonCannon:
-	ValidTargets: Ground, Air
+	ValidTargets: Ground, Water, Air
```

Comment:
TS doesn't use assign `Water` TargetType to anything, so the changes here aren't needed...?